lib/core/utils.js

```diff
@@ -361,11 +361,20 @@ RESUtils.thingsContainer = function(body) {
 
 	return container;
 };
-RESUtils.things = function(container) {
+RESUtils.$things = function(container) {
 	container = container || RESUtils.thingsContainer();
 	return $(container).find(RESUtils.thing.prototype.thingSelector);
 };
+RESUtils.things = function(container) {
```

Comment:
Do we ever need `$things`? I feel like we'd only ever need the array of the elements all wrapped (though I guess I didn't look too closely to see where we used it in code).